WordPress/Classes/Services/BlogService.m

```diff
@@ -159,9 +159,14 @@ - (void)syncBlogsForAccount:(WPAccount *)account
 {
     DDLogMethod();
 
+    NSManagedObjectID *accountID = account.objectID;
     id<AccountServiceRemote> remote = [self remoteForAccount:account];
     [remote getBlogsWithSuccess:^(NSArray *blogs) {
         [self.managedObjectContext performBlock:^{
+            WPAccount *account = (WPAccount *)[self.managedObjectContext existingObjectWithID:accountID error:nil];
+            if (!account) {
+                return;
```

Comment:
>  I'm beginning to think passing any parameters by NSManagedObjectID is going to have to be the requirement.

@astralbodies I think that's probably for the best, but I think we should do a second PR for that and try to get this into 5.4, since my goal was to stop crashing when a site is deleted and a call was in progress